README.md

```diff
@@ -1,36 +1,83 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+<img src = "https://github.com/fiizzy/calentre/blob/main/doc/images/logo.png?raw=true"  >
 
-## Getting Started
+# Calentre 📅💰
 
-First, run the development server:
+Welcome to Calentre, the open-source alternative to Calendly! 🌟 Manage your appointments, get paid, and enjoy the scheduling experience.
```

Comment:
"Welcome to the Client (invitee) side of Calentre. This repo contains the code that allows on invitee (client) to book a meeting based on the schedule of the admin (meeting owner) "